### PR TITLE
Add Readme users

### PR DESCRIPTION
**Pull request by ealexisaraujo**

### CODE REVIEW

File: components/users/README.md [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+```js
+const usersService = new UsersService();
+
+const { createUserId: id, username } = await usersService.createUser({
+  documentID: 21873837287,
+  firstName: 'Alexis',
+  lastName: 'Araujo',
+  email: 'ala1289@gmail.com',
+  contactNumber: 7223228383,
+  typeOfUser: 'Administrator',
```
Comment (hyfi06) [inline components/users/README.md:348]: :eyes:

File: components/users/README.md [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+const { createUserId: id, username } = await usersService.createUser({
+  documentID: 21873837287,
+  firstName: 'Mayne',
+  lastName: 'Snasel',
+  email: 'msnasel0@tripadvisor.com',
+  contactNumber: 624215528148,
+  typeOfUser: 'Administrator',
+});
+
+console.log((createUserId: id), username); // 5ed063fd50185b52002f0570 , alexis.araujo.0001
```
Comment (hyfi06) [inline components/users/README.md:372]: :eyes:

File: components/users/README.md [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+        "lastName": "Snasel",
+        "email": "msnasel0@tripadvisor.com",
+        "contactNumber": 624215528148,
+        "typeOfUser": "Administrator",
+        "username": "mayne.snasel.4171",
+        "createdAt": "2020-05-28T15:36:57.064Z",
+        "updatedAt": "2020-05-28T17:58:55.489Z"
+    },
+    "message": "user retrieved"
+}
```
Comment (hyfi06) [inline components/users/README.md:271]: it is a route response, not of service

File: components/users/README.md [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+            "_id": "5ec5ce16fc13ae1506000064",
+            "isActive": false,
+            "documentID": 21873837287,
+            "firstName": "Mayne",
+            "lastName": "Snasel",
+            "typeOfUser": "Administrator",
+            "username": "mayne.snasel.4171",
+    ],
+    "message": "users listed"
+*/
```
Comment (hyfi06) [inline components/users/README.md:299]: it is a route response, not of service

File: components/users/README.md [hunk 5]
```diff
@@ -0,0 +1,373 @@
...
+
+Get all users by name or typeOfUser or documentID or isActive.
+
+Result is a array that contains objects with users data.
+
+```js
+const userService = new UsersService();
+
+const users = await usersService.getUsers(
+  { name: 'alexis' } || { typeOfUser: 'Administrator' } || {
```
Comment (hyfi06) [inline components/users/README.md:312]: :eyes:

File: components/users/README.md [hunk 6]
```diff
@@ -0,0 +1,373 @@
...
+        "username": "mayne.snasel.4171",
+        "typeOfUser": "Administrator",
+        "isActive": false,
+        "firstName": "Mayne",
+        "lastName": "Snasel",
+        "defaultPath": "/"
+    }
+}
+*/
+```
```
[the remark this answers is not in the document]
Comment (ealexisaraujo) [reply components/users/README.md]: Done!

File: components/users/README.md (hunk 2 above)
Comment (ealexisaraujo) [reply components/users/README.md:373]: Fixed!

File: components/users/README.md (hunk 3 above)
Comment (ealexisaraujo) [reply components/users/README.md]: Fixed!

File: components/users/README.md (hunk 4 above)
Comment (ealexisaraujo) [reply components/users/README.md]: Fixed!

File: components/users/README.md (hunk 5 above)
Comment (ealexisaraujo) [reply components/users/README.md]: Fixed!

File: components/users/README.md [hunk 7]
```diff
@@ -0,0 +1,373 @@
...
+```js
+const userService = new UsersService();
+
+const user = await userService.getUserId({ '5ec5ce16fc13ae1506000064' });
+
+console.log(user);
+/*
+{
+    "data": {
+        "_id": "5ec5ce16fc13ae1506000064",
```
[the remark this answers is not in the document]
Comment (ealexisaraujo) [reply components/users/README.md]: Fixed!

**Pull request merged**